assets/styles.css

```diff
@@ -0,0 +1,202 @@
+/* Apple Clone Website Styles */
+
+@import url("https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400&display=swap");
+
+/* CSS RESET */
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+a {
+  text-decoration: none;
+}
+
+a:hover {
+  text-decoration: underline;
+}
+
+ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  width: 100%;
+  align-items: center;
+}
+
+h3 {
+  margin: 20px auto;
+}
+
+.container {
+  width: 100%;
+  max-width: 1440px;
+  margin: auto;
+  font-family: "Lato", sans-serif;
+}
+
+.flex {
+  display: flex;
+}
+
+.color-white {
+  color: #fff;
+}
+
+.hero {
+  min-height: 77vh;
+  background-image: url("./images/your_verse_hero.jpg");
+  background-repeat: no-repeat;
+  background-position: 50% 100%;
+  background-size: 1440px 1315px;
+}
+
+.hero-main {
+  text-align: center;
+  padding-top: 40px;
+}
+
+.nav-items {
+  width: 900px;
+  margin: 0 auto;
+  align-items: center;
+  justify-content: center;
+  border-radius: 12px;
+  padding: 0 10px;
+  position: relative;
+  text-align: center;
+  top: 3vh;
+  background: rgb(144, 144, 144);
+  background:
+    linear-gradient(
+      180deg,
+      rgba(144, 144, 144, 1) 0%,
+      rgba(51, 51, 51, 1) 50%,
+      rgba(133, 133, 133, 1) 82%
+    );
+}
+
+.nav-item {
+  color: #fff;
+  line-height: 3;
+  font-size: 14px;
+  flex: 1;
+  text-align: center;
+  background: rgb(144, 144, 144);
+  background:
+    linear-gradient(
+      180deg,
+      rgba(144, 144, 144, 1) 0%,
+      rgba(51, 51, 51, 1) 50%,
+      rgba(133, 133, 133, 1) 82%
+    );
+  border-right: solid thin rgba(255, 255, 255, 0.15);
+  padding: 0 10px;
+}
+
+.nav-item:first-child {
+  padding-top: 5px;
+}
+
+.nav-item:last-child {
+  border: none;
+}
+
+.nav-item a {
+  color: #fff;
+  height: 32px;
+}
+
+.nav-item a:hover {
+  text-decoration: none;
+}
+
+.nav-link:hover {
+  cursor: pointer;
+  background: rgb(103, 102, 102);
+  background:
+    linear-gradient(
+      180deg,
+      rgba(103, 102, 102, 1) 0%,
+      rgba(51, 51, 51, 1) 50%,
+      rgba(88, 88, 88, 1) 82%
+    );
+}
+
+form {
+  position: relative;
+}
+
+form span {
+  position: absolute;
+  left: 8px;
+}
+
+form input {
+  padding: 3px;
+  padding-left: 24px;
+  color: #333;
+  background: transparent;
+  transition: 0.5s;
+  border-radius: 14px;
+  width: 120px;
+}
+
+form input:focus {
+  outline: none;
+  background-color: #fff;
+  width: 230px;
+}
+
+form input:focus + span {
+  color: #333;
+}
+
+.fa.fa-play-circle {
+  font-size: 40px;
+}
+
+/* Products section */
+
+.products {
+  min-height: 22vh;
+}
+
+.product {
+  width: 100%;
+}
+
+/* footer section */
+.main {
+  width: 960px;
+  margin: 15px auto;
+  font-size: 11px;
+}
+
+ul li {
+  text-align: center;
+  padding: 0 5px;
+  display: inline;
+  border-right: solid thin #333;
```

Comment:
- [ ] Please take out that border-right for your bottom images, as well as the padding at the beggining and the ending of the row. You also need a space on the top to emulate the original (I know you are using this styling for your footer li elements, how about you add a class specially for them with these rules?)
![image](https://user-images.githubusercontent.com/53256139/88643872-dde29d80-d087-11ea-9c92-250e83d5b389.png)